examples/in_range.rs

```diff
@@ -0,0 +1,156 @@
+use aery::prelude::Up;
+use aery::prelude::*;
+use bevy::log;
+use bevy::prelude::*;
+use std::fmt::Display;
+
+#[derive(Component)]
+struct MovingEntity {
+    direction: f32, // Movement direction along the x-axis, positive or negative.
+}
+
+#[derive(Component)]
+struct Name(&'static str);
+
+impl Display for Name {
```

Comment:
I'd just slap a `Deref` derive on the component or use `.0` instead when accessing. No need to noisy up examples with boilerplate.

Reply:
reworked example to be more readable and less noisy